packages/wallet-sdk/src/derive.ts

```diff
@@ -0,0 +1,102 @@
+import { BIP32Interface } from 'bip32';
+import { Buffer } from 'buffer';
```

Comment:
Usage of the NodeJS `buffer` has been replaced with a runtime agnostic lib, see example usage here:
https://github.com/blockstack/stacks.js/blob/8e99b763fbbd486cce87749e9e1b21015c61e946/packages/transactions/src/clarity/serialize.ts#L1

This allows the libs to work in both Node.js, browsers, and other runtimes without requiring extra bundling/polyfill utils.

Can you update usages to this convention?

Reply:
@zone117x updated these usages to follow the convention.